COMMUNITY-ROLES.md

```diff
@@ -89,108 +89,129 @@ using one of the template messages below as a base.
 
 ### For adding new collaborators
 
-Open an issue with the following message:
-
-> Hi, \@username(s)! You seem to be enjoying contributing to the tldr-pages project.
-You now have had five distinct pull requests merged (\[LINKS TO THE RELEVANT PRs]),
-which qualifies you to become a collaborator in this repository,
-as explained in our
-\[community roles documentation](https://github.com/tldr-pages/tldr/blob/master/COMMUNITY-ROLES.md).
->
-> As a collaborator, you will have commit access
-and can therefore merge pull requests from others, label and close issues,
-and perform various other maintenance tasks that are needed here and there.
-Of course, all of this is voluntary
-— you're welcome to contribute to the project in whatever ways suit your liking.
->
-> If you do decide to start performing maintenance tasks, though,
-we only ask you to get familiar with the
-\[maintainer's guide](https://github.com/tldr-pages/tldr/blob/master/contributing-guides/maintainers-guide.md).
->
-> Thanks for all your work so far!
-
-Once they acknowledge the message,
-go to https://github.com/tldr-pages/tldr/settings/collaboration
-and add them to the repository as collaborator with write permissions.
+1. Open an issue with the following message template (edit it as appropriate):
+
+   ```
+   Hi, @username! You seem to be enjoying contributing to the tldr-pages project.
+   You now have had five distinct pull requests merged ([LINKS TO THE RELEVANT PRs]),
+   which qualifies you to become a collaborator in this repository,
+   as explained in our
+   [community roles documentation](https://github.com/tldr-pages/tldr/blob/master/COMMUNITY-ROLES.md).
+
+   As a collaborator, you will have commit access
+   and can therefore merge pull requests from others, label and close issues,
+   and perform various other maintenance tasks that are needed here and there.
+   Of course, all of this is voluntary
+   — you're welcome to contribute to the project in whatever ways suit your liking.
+
+   If you do decide to start performing maintenance tasks, though,
+   we only ask you to get familiar with the
+   [maintainer's guide](https://github.com/tldr-pages/tldr/blob/master/contributing-guides/maintainers-guide.md).
+
+   Thanks for all your work so far!
+   ```
+
+2. Once they acknowledge the message,
```

Comment:
```suggestion
2. Once they acknowledge the message, if they accept,
```

I'd specify that one should actually accept the proposed role change (same goes for the other sections, except the last one probably).

Reply:
Seems logical 😋

Reply:
Yeah, makes sense :)